controllers/questionsController.js

```diff
@@ -0,0 +1,60 @@
+const models = require('../models');
+const questions = require('../models/questions');
+const app = require('../app');
+
+//Post a question
+const addQuestion = async (req, res) => {
+    const { title, body } = req.body
```

Comment:
- Validate request payload
- Change to `const { title, question } = req.body` instead